canu/2.0/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Use ubuntu as base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="Canu"
+LABEL software.version="2.0; 2.17; 1.4.3"
+LABEL description="Long read assembly and polishing tools"
+LABEL website="https://github.com/marbl/canu"
+LABEL license="See program repositories"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# install python
+RUN apt-get update && apt-get install -y \
+   perl \
+   default-jre \
+   gnuplot \
+   libgcc-5-dev \
+   libgomp1 \
+   maven \
+   git \
+   wget \
+   python3 \
+   build-essential \
+   cmake \
+   zlib1g-dev && \
+   apt-get clean
```

Comment:
I learned that `apt-get clean` is not necessary to run since offical ubuntu docker images run it (without you knowing). I would replace that command with:
`apt-get autoclean && rm -rf /var/lib/apt/lists/*`

modeled after docker's best practice list: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run